### ISSUE TITLE
Missing firmware for Thinkpad T495  -- 1.31   (R12UL61W)   (2022-09-01)

### ISSUE THREAD

**Issue by speachy** [fwupd/firmware-lenovo#272]
https://download.lenovo.com/pccbbs/mobiles/r12ul61w.zip
https://download.lenovo.com/pccbbs/mobiles/r12ul61w.txt

Interestingly, they provide a direct Linux update cab in that zip file, but when trying 'fupdmgr install XXX.cab' it claims 'No supported devices found' This is with fwupd 1.8.5 on Fedora 36.

**Comment by speachy** [comment]
```
Handle 0x000E, DMI type 0, 24 bytes
BIOS Information
        Vendor: LENOVO
        Version: R12ET60W(1.30 )
        Release Date: 02/16/2022
        Address: 0xE0000
```

```
$ fwupdmgr install /tmp/Lenovo-ThinkPad-T495-AMD-SystemFirmware-1.31W.cab
Decompressing…           [       \                               ]
No supported devices found
```

Also, I noticed that this one's internal version string is "0.1.31" but the current (v1.30) firmware's reported string is "0.1.36" which might be part of the problem.  fwupdmgr's output:

```
├─System Firmware:
│ │   Device ID:          a45df35ac0e948ee180fe216a5f703f32dda163f
│ │   Summary:            UEFI ESRT device
│ │   Current version:    0.1.36
│ │   Vendor:             Lenovo (DMI:LENOVO)
│ │   Update State:       Success
│ │   Update Message:     The computer will be restarted automatically after updating BIOS completely. Do NOT turn off your computer or remove the AC adaptor while update is in progress.
│ │   GUIDs:              b3ddeb3b-7264-43f0-bf36-a2b108adc70a
```
And the .cab's xml file:
```
  <description>
    <p>
      Lenovo ThinkPad T495 System Firmware R12ET61W, version 1.31
    </p>
  </description>
[...]
  <releases>
    <release version="0.1.31" timestamp="1628582255">
      <description>
        <p>Lenovo ThinkPad T495 System Firmware</p>
      </description>
    </release>
  </releases>

**Comment by mrhpearson** [comment]
It is on LVFS - but under embargo and doesn't look like they have done the testing on it yet :( 
I've flagged this to the FW team (internal ticket  LO-2058).

**Comment by speachy** [comment]
Well, it's publicly posted on the T495's download site.  :)

https://pcsupport.lenovo.com/us/en/products/laptops-and-netbooks/thinkpad-t-series-laptops/thinkpad-t495-type-20nj-20nk/20nj/20njcto1ww/pf21vbhd/downloads/ds539877-bios-update-utility-bootable-cd-for-windows-10-64-bit-thinkpad-t495?category=BIOS%2FUEFI

(Come to think of it, the v1.30 firmware I got from LVFS last time never got posted to the download site, so.. methinks lenovo has some internal process hiccups going on..)

Thanks for escalating this!

**Comment by ChiWei-Chen** [comment]
Hi @speachy , I work with @mrhpearson.
The FW steam seems released the new BIOS 1.37 to stable, please let me know if this doesn't work for you. Thanks!
https://fwupd.org/lvfs/devices/com.lenovo.ThinkPadR12ETXXW.firmware
https://fwupd.org/lvfs/devices/com.lenovo.ThinkPadR12ETXXP.firmware

**Comment by speachy** [comment]
I just installed the update through fwupd, and it seems okay.  Thanks for getting this handled so quickly!
